lib/generic/genericIcon.dart

```diff
@@ -64,4 +65,13 @@ class GenericIcon {
           ),
         ));
   }
+  static Widget buildGenericSpinner() {
```

Comment:
![grafik](https://user-images.githubusercontent.com/6030353/41421206-83a64b6c-6ff6-11e8-9fed-b2e736c8fd18.png)

Reply:
My Fault. Sean told me that this is because if didnt fetch new dependencies.